source/mongodb-handshake/handshake.rst

```diff
@@ -70,14 +70,18 @@ Connection handshake
 --------------------
 
 MongoDB uses the ``hello`` or ``isMaster`` commands for handshakes and topology
-monitoring. ``hello`` is the modern and preferred command. ``isMaster`` is referred
-to as "legacy hello" and is maintained for backwards compatibility with servers
+monitoring. ``hello`` is the modern and preferred command. ``hello`` must
+always be sent using the ``OP_MSG`` protocol. ``isMaster`` is referred to as
+"legacy hello" and is maintained for backwards compatibility with servers
 that do not support the ``hello`` command.
 
 If a `server API version <../versioned-api/versioned-api.rst>`__ is requested,
-drivers MUST use the ``hello`` command for the initial handshake. If server API
-version is not requested, drivers MUST use legacy hello for the initial handshake
-and include ``helloOk: true`` in the handshake request.
+drivers MUST use the ``hello`` command for the initial handshake and use the
+``OP_MSG`` protocol. If server API version is not requested, drivers MUST
```

Comment:
Note that `loadBalanced: true` also indicates that you must be talking to a load balancer that supports `OP_MSG`.

Reply:
Done.

Reply:
I don't think this is true. Can you point to where a spec says that load balanced should use hello+OP_MSG?

Reply:
This was discussed with the team during stand-up. The decision made by @behackett was to make this change but in a separate ticket [DRIVERS-2033](https://jira.mongodb.org/browse/DRIVERS-2033)